### PR TITLE
fix: keep lockfile up to date example windows regression

### PR DESCRIPTION
**Pull request by kormide**
This is the last windows regression aside from the `node-gyp` issue.

### CODE REVIEW

File: js/repositories.bzl [hunk 1]
```diff
@@ -38,7 +38,7 @@ def rules_js_dependencies():
     maybe(
         http_archive,
         name = "aspect_bazel_lib",
-        sha256 = "2db5b7176459c23f35a6cd45ff466fc3784bb17fdfa17a9bfeb1ac837796464c",
-        strip_prefix = "bazel-lib-1.2.0",
-        url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v1.2.0.tar.gz",
+        sha256 = "a62838a8bb32478db2cf557b59279e566d628e064f91071daf1986c32463c2c9",
+        strip_prefix = "bazel-lib-1.2.1",
+        url = "https://github.com/aspect-build/bazel-lib/archive/refs/tags/v1.2.1.tar.gz",
```
Comment (kormide) [inline js/repositories.bzl:43]: Needed for the copy_to_bin same file different target fix.

File: examples/assert_lockfile_up_to_date/BUILD [hunk 2]
```diff
@@ -12,12 +12,18 @@ js_test(
         ":pnpm-lock.yaml",
     ],
     entry_point = "assert_lockfile_frozen.js",
+    env = {
+        "APPDATA": "unused_but_pnpm_fails_if_not_set_on_windows"
```
Comment (gregmagolan) [inline examples/assert_lockfile_up_to_date/BUILD:16]: what is this black magic?

Comment (kormide) [reply examples/assert_lockfile_up_to_date/BUILD:16]: :mage_man:

[the remark this answers is not in the document]
Comment (gregmagolan) [reply examples/assert_lockfile_up_to_date/BUILD:16]: Can you add a comment with some more context for future reference?

**Review by gregmagolan — APPROVED** [review]

File: examples/assert_lockfile_up_to_date/BUILD (hunk 2 above)
[the remark this answers is not in the document]
Comment (kormide) [reply examples/assert_lockfile_up_to_date/BUILD]: Added

**Pull request merged**